### PR TITLE
Pgebheim/ch14959/update search functionality

### PR DESCRIPTION
**Pull request by bthaile**
https://app.clubhouse.io/augur/story/14959/update-search-functionality

split out search fields.

### CODE REVIEW

**Comment by bthaile** [comment]
What about the old table load? Does that stay?

**Comment by bthaile** [comment]
Updated to leave migration files, just removed the first inserts in the old search_en table.

File: src/utils/content-search-builder.ts [hunk 1]
```diff
@@ -1,17 +1,29 @@
...
+  } = marketsDataToInsert;
+
+  return {
+    marketId,
+    category,
+    tags: filteredTags.length > 0 ? filteredTags : "",
+    shortDescription: shortDescription !== null ? marketsDataToInsert.shortDescription : "",
+    longDescription: longDescription !== null ? marketsDataToInsert.longDescription : "",
+    resolutionSource: resolutionSource !== null ? marketsDataToInsert.resolutionSource : "",
+    scalarDenomination: scalarDenomination != null ? marketsDataToInsert.scalarDenomination : "",
```
Comment (epheph) [inline src/utils/content-search-builder.ts]: There's some inconsistent checks here, `!=` vs `!==` . We probably want the looser version, `!=`. I often prefer the even more loose/shorter:

```
marketsDataToInsert.scalarDenomination || ""
```
any false-y value gets empty string, in fewer lines

File: src/blockchain/log-processors/market-created.ts [hunk 2]
```diff
@@ -104,7 +101,7 @@ export function processMarketCreatedLog(db: Knex, augur: Augur, log: FormattedEv
               db.insert(marketsDataToInsert).into("markets").asCallback(next);
             },
             (next: AsyncCallback): void => {
-              db.raw("insert into search_en(marketId, content) values( ?, ? )", [fullTextStringInsert.marketId, fullTextStringInsert.content]).asCallback(next);
+              db.insert(marketSearchDataToInsert).into("search_en").asCallback(next);
```
Comment (epheph) [inline src/blockchain/log-processors/market-created.ts]: Much better format here 👍

**Pull request merged**